django/core/mail/backends/smtp.py

```diff
@@ -18,37 +18,25 @@ class EmailBackend(BaseEmailBackend):
 
     def __init__(
         self,
-        host=None,
-        port=None,
-        username=None,
-        password=None,
-        use_tls=None,
         fail_silently=False,
-        use_ssl=None,
-        timeout=None,
-        ssl_keyfile=None,
-        ssl_certfile=None,
-        **kwargs,
+        use_localtime=False,
+        **options,
```

Comment:
I think it would be better to keep the individual args listed, rather than collapsing them all into `**options`. Individual args are clearer for other people reading the code: e.g., to find out what they can include in settings OPTIONS or pass as overrides to get_connection(). (And `**options` complicates efforts like django-stubs.)